### PR TITLE
fix gov & guardian role in deployment

### PR DESCRIPTION
**Pull request by orbxball**

### CODE REVIEW

File: scripts/deploy.py [hunk 1]
```diff
@@ -62,12 +62,12 @@ def main():
         gov_default = dev.address
     else:
         gov_default = "ychad.eth"
-    gov = get_address("Yearn Governance", default=gov_default)
+    gov = get_address("Yearn Governance", default="0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7")
```
Comment (fubuloubu) [inline scripts/deploy.py:65]: What is this address?

Comment (orbxball) [reply scripts/deploy.py:65]: strategist multisig.
these two changes are to match the latest version of `OPERATION.md`

File: scripts/deploy.py [hunk 2]
```diff
@@ -62,12 +62,12 @@ def main():
...
 
     rewards = get_address(
         "Rewards contract", default="0x93A62dA5a14C80f265DAbC077fCEE437B1a0Efde"
```
Comment (fubuloubu) [inline scripts/deploy.py:68]:
```suggestion
        "Rewards contract", default="treasury.ychad.eth"
```

File: scripts/deploy.py (hunk 1 above)
Comment (fubuloubu) [reply scripts/deploy.py:65]: Please add an ENS entry for this msig and reference that instead

Comment (orbxball) [reply scripts/deploy.py:65]: I don think we have ENS on strategist msig. But it's fine. We agreed to set `gov = dev.ychad.eth` on experimental vault.

Comment (fubuloubu) [reply scripts/deploy.py:65]: that's a big change, to me... I was expecting strategists would gov experiments because the whole idea is that experiments drive production strategies and vaults. devs are in charge of the production protocol, strategists in charge of experiments.

Comment (fameal) [reply scripts/deploy.py]: This was a long discussion and we finally agreed on ydev governs production vaults. Other stuff is up to strategists. We can keep this as it was.

File: scripts/deploy.py [hunk 3]
```diff
@@ -62,12 +62,12 @@ def main():
         gov_default = dev.address
     else:
         gov_default = "ychad.eth"
-    gov = get_address("Yearn Governance", default=gov_default)
+    gov = get_address("Yearn Governance", default="dev.ychad.eth")
```
Comment (fubuloubu) [inline scripts/deploy.py:65]: Revert this change back to `gov_default`

**Review by fubuloubu — APPROVED** [review]

File: scripts/deploy.py [hunk 4]
```diff
@@ -62,12 +62,12 @@ def main():
...
 
     rewards = get_address(
-        "Rewards contract", default="0x93A62dA5a14C80f265DAbC077fCEE437B1a0Efde"
+        "Rewards contract", default="treasury.ychad.eth"
```
Comment (fubuloubu) [inline scripts/deploy.py:68]: Run `black .` on this

**Comment by fubuloubu** [comment]
Please rebase on `master`

File: scripts/deploy.py [hunk 5]
```diff
@@ -59,15 +59,15 @@ def main():
     token = Token.at(get_address("ERC20 Token"))
 
     if use_proxy:
-        gov_default = dev.address
+        gov_default = (
+            "0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7"  # strategist msig, no ENS
```
Comment (fubuloubu) [inline scripts/deploy.py:63]: can the strategist msig be added to ENS? e.g. `brains.ychad.eth`

Comment (orbxball) [reply scripts/deploy.py:63]: It's on Klim's long list already.
